core/templates/dev/head/pages/exploration-player-page/services/learner-answer-info.service.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to ask learner for answer info.
+ *
+ */
+
+require(
+  'pages/exploration-player-page/services/exploration-engine.service.ts');
+
+var oppia = require('AppInit.ts').module;
+
+oppia.factory('LearnerAnswerInfoService', [
+  'ExplorationEngineService',
```

Comment:
is this used anywhere?

Reply:
Yeah, I am using ExplorationEngineService